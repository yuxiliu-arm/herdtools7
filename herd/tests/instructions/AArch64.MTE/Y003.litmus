AArch64 Y003
(* IRG with filtering *)
{
int x=100;
0:X0=x;
}
 P0            ;
 MOV W2,#3     ; (* 0b11 *)
 IRG X1,X0,X2  ;
forall (not (0:X1=x:t0) /\ not (0:X1=x:t1))

