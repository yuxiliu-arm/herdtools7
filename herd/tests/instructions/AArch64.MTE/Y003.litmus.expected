Test Y003 Required
States 14
0:X1=x:t10;
0:X1=x:t11;
0:X1=x:t12;
0:X1=x:t13;
0:X1=x:t14;
0:X1=x:t15;
0:X1=x:t2;
0:X1=x:t3;
0:X1=x:t4;
0:X1=x:t5;
0:X1=x:t6;
0:X1=x:t7;
0:X1=x:t8;
0:X1=x:t9;
Ok
Witnesses
Positive: 14 Negative: 0
Condition forall (not (0:X1=x:t0) /\ not (0:X1=x:t1))
Observation Y003 Always 14 0
Hash=5636d8f61ba6e9c72c60cc386646da79

